src/sentry/api/endpoints/organization_spans_frequency_stats.py

```diff
@@ -0,0 +1,99 @@
+from google.protobuf.json_format import MessageToDict
+from rest_framework.exceptions import ParseError
+from rest_framework.request import Request
+from rest_framework.response import Response
+from sentry_protos.snuba.v1.endpoint_trace_item_stats_pb2 import (
+    AttributeDistributionsRequest,
+    StatsType,
+    TraceItemStatsRequest,
+)
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import NoProjects, OrganizationEventsV2EndpointBase
+from sentry.api.endpoints.organization_spans_fields import OrganizationSpansFieldsEndpointSerializer
+from sentry.models.organization import Organization
+from sentry.search.eap.resolver import SearchResolver
+from sentry.search.eap.span_columns import SPAN_DEFINITIONS
+from sentry.search.eap.types import SearchResolverConfig
+from sentry.snuba.referrer import Referrer
+from sentry.utils import snuba_rpc
+
+
+@region_silo_endpoint
+class OrganizationSpansFrequencyStatsEndpoint(OrganizationEventsV2EndpointBase):
+    snuba_methods = ["GET"]
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
```

Comment:
Let's create a ticket to document this API as a follow up! This API should likely be a public one eventually so it would be good to document it while you have the context!